index_documents.go

```diff
@@ -48,7 +55,7 @@ func (i Index) GetDocuments(request *DocumentsRequest, resp interface{}) error {
 	return nil
 }
 
-func (i Index) AddDocuments(documentsPtr interface{}, primaryKey ...string) (resp *AsyncUpdateID, err error) {
+func (i Index) addDocuments(documentsPtr interface{}, contentType string, primaryKey ...string) (resp *AsyncUpdateID, err error) {
```

Comment:
It could be nice to rename this function `AddDocumentsRaw` and make it public?

Reply:
In that case I think we'll have to document which Content-Types are accepted (or limit the possible values with a custom type and predefined constants) and how the different document types are handled by `sendRequest`, otherwise there might be confusion about the method.

Do you have a specific use case in mind?

Reply:
Not really. You are right it will be easier to leave it private but if it is possible to rename it to avoid confusion?